client/my-sites/themes/single-site.jsx

```diff
@@ -40,6 +40,7 @@ const SingleSiteThemeShowcaseWithOptions = ( props ) => {
 					'purchase',
 					'activate',
 					'tryandcustomize',
+					'upload',
```

Comment:
Adding 'upload' to the list here means we get it as an option on the ... button for individual themes, which I don't think we want:
<img width="311" alt="screen shot 2016-12-13 at 17 10 50" src="https://cloud.githubusercontent.com/assets/7767559/21150549/5153835e-c157-11e6-973e-35ec3ab8235b.png">